pkg/tests/cross-tests/input_cross_test.go

```diff
@@ -217,3 +217,35 @@ func TestInputsEmptyConfigModeAttrSet(t *testing.T) {
 		SkipCompareRawConfig: true,
 	})
 }
+
+func TestExplicitNilList(t *testing.T) {
+	skipUnlessLinux(t)
+	// This is an explicit null on the tf side:
+	// resource "crossprovider_testres" "example" {
+	//     f0 = null
+	// }
+	// This is different to an unspecified value.
+	// We might not be able to distinguish between null and missing collections.
+	runCreateInputCheck(t, inputTestCase{
+		Resource: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"f0": {
+					Optional: true,
+					Type:     schema.TypeList,
+					Elem: &schema.Schema{
+						Type:     schema.TypeMap,
+						Optional: true,
+						Computed: true,
+						Elem: &schema.Schema{
+							Type:      schema.TypeInt,
+							Optional:  true,
+							Sensitive: true,
+						},
+					},
+				},
+			},
+		},
+		// TODO: How does one express this with tftypes?
```

Comment:
`tftypes.NewValue(typ, nil)` expresses a null value. Every type is inhabited by a null value.

Reply:
Thank you for the comment, I tried it again and it is indeed the case - I must have got something wrong when I tried it last time.